runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersConfig.java

```diff
@@ -200,6 +200,25 @@ default boolean showCannon()
 		return true;
 	}
 
+	// Stored Data
+	@ConfigItem(
+			keyName = "cannonWorld",
+			name = "",
+			description = "",
+			hidden = true
+	)
```

Comment:
Too much indent above, should probably review the following article:
https://github.com/runelite/runelite/wiki/Checkstyle-with-IntelliJ-IDEA